plugins/wallabag.koplugin/main.lua

```diff
@@ -0,0 +1,692 @@
+                        keep_menu_open = true,
+                        callback = function(touchmenu_instance)
+                            self:setDownloadDirectory(touchmenu_instance)
+                        end,
+                    },
+                    {
+                        text_func = function()
+                            local filter
+                            if not self.filter_tag or self.filter_tag == "" then
+                                filter = "All articles"
+                            else
+                                filter = self.filter_tag
+                            end
+                            return _("Filter articles by tag") .. " (" .. filter .. ")"
+                        end,
+                        keep_menu_open = true,
+                        callback = function(touchmenu_instance)
+                            self:setFilterTag(touchmenu_instance)
+                        end,
+                    },
+                    {
+                        text = _("Remotely delete Finished articles"),
+                        checked_func = function() return self.is_delete_finished end,
+                        callback = function()
+                            self.is_delete_finished = not self.is_delete_finished
+                            self:saveSettings()
+                        end,
+                    },
+                    {
+                        text = _("Remotely delete 100% read articles"),
+                        checked_func = function() return self.is_delete_read end,
+                        callback = function()
+                            self.is_delete_read = not self.is_delete_read
+                            self:saveSettings()
+                        end,
+                    },
+                    {
+                        text = _("Process deletions when downloading"),
+                        checked_func = function() return self.is_auto_delete end,
+                        callback = function()
+                            self.is_auto_delete = not self.is_auto_delete
+                            self:saveSettings()
+                        end,
+                    },
+                    {
+                        text = _("Synchronise remotely deleted files"),
+                        checked_func = function() return self.is_sync_remote_delete end,
+                        callback = function()
+                            self.is_sync_remote_delete = not self.is_sync_remote_delete
+                            self:saveSettings()
+                        end,
+                    },
+                    {
+                        text = _("Help"),
+                        keep_menu_open = true,
+                        callback = function()
+                            UIManager:show(InfoMessage:new{
+                                text = _('Download directory: use a directory that is exclusively used by the Wallabag plugin. ' ..
```

Comment:
Please use a multiline comment. Our translation string extraction regex will fail on this.

Reply:
Is a max of 79 chars OK or should I split them further?